### DISCUSSION TITLE
Sherlock Installed in "not in a path"

### DISCUSSION THREAD

**Discussion by mahir793** [sherlock-project/sherlock#2217] [Q&A] [answered]
![Screenshot 2024-07-03 083701](https://github.com/sherlock-project/sherlock/assets/153705161/9f85fb0d-5d4d-4d60-a135-5cb48eda4a8e)
![Screenshot 2024-07-03 083716](https://github.com/sherlock-project/sherlock/assets/153705161/138b6a30-5723-4b9a-90e3-60a662b1d713)
These two are happening and i can't use sherlock. what to do? please help

**Comment by ppfeister** [comment, marked as answer]
`sherlock` as a single command won't work until you add pip's userspace bin to your system PATH.

Assuming `echo $SHELL` is /usr/bin/zsh...

```bash
echo 'export PATH="$PATH:/home/mahir793/.local/bin"' >> ~/.zshrc
source ~/.zshrc
```

Assuming `echo $SHELL` is /usr/bin/bash...

```bash
echo 'export PATH="$PATH:/home/mahir793/.local/bin"' >> ~/.bashrc
source ~/.bashrc
```

Then you can run `sherlock` (or anything else you install with pip) normally. (`sherlock`, not `pip sherlock`)

**Reply by mahir793** [reply]
Thanks a lot. It worked.
